x-pack/plugins/security_solution/server/endpoint/lib/artifacts/task.ts

```diff
@@ -44,22 +48,37 @@ export class ManifestTask {
   constructor(setupContract: ManifestTaskSetupContract) {
     this.endpointAppContext = setupContract.endpointAppContext;
     this.logger = this.endpointAppContext.logFactory.get(this.getTaskId());
+    const { packagerTaskInterval, packagerTaskTimeout, packagerTaskPackagePolicyUpdateBatchSize } =
+      this.endpointAppContext.serverConfig;
+
+    this.logger.info(
+      `Registering ${ManifestTaskConstants.TYPE} task with timeout of [${packagerTaskTimeout}], interval of [${packagerTaskInterval}] and policy update batch size of [${packagerTaskPackagePolicyUpdateBatchSize}]`
+    );
 
     setupContract.taskManager.registerTaskDefinitions({
       [ManifestTaskConstants.TYPE]: {
         title: 'Security Solution Endpoint Exceptions Handler',
-        timeout: ManifestTaskConstants.TIMEOUT,
+        timeout: packagerTaskTimeout,
         createTaskRunner: ({ taskInstance }: { taskInstance: ConcreteTaskInstance }) => {
           return {
             run: async () => {
-              const taskInterval = (await this.endpointAppContext.config()).packagerTaskInterval;
-              const startTime = new Date().getTime();
+              const taskInterval = packagerTaskInterval;
+              const startTime = new Date();
+
+              this.logger.info(`Started. Checking for changes to endpoint artifacts`);
+
               await this.runTask(taskInstance.id);
+
               const endTime = new Date().getTime();
-              this.logger.debug(
-                `${ManifestTaskConstants.TYPE} task run took ${endTime - startTime}ms`
+
+              this.logger.info(
```

Comment:
Should this be at `debug` level? otherwise, it could be logging this timing every minute (by default) even there is nothing to do on the task

Reply:
I changed it to `.info` so that we can always get information about how long the task actually took to run without having ot ask a customer to set it debug level. This (IMO) is helpful, especially since we don't have `.cancel()` logic, to understand how long these tasks are actually running at customer's env.